src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -34,19 +37,24 @@
 {
     const ROUTE_NAME_PREFIX = 'api_';
     const DEFAULT_ACTION_PATTERN = 'api_platform.action.';
+    const SUBCOLLECTION_SUFFIX = '_get_subcollection';
```

Comment:
Indeed I did that on purpose because we only support `get` for now. It's better hardcoded here then below IMO.